Harvest.Core/Domain/Invoice.cs

```diff
@@ -19,11 +19,17 @@ public class Invoice
         public decimal Total { get; set; }
 
         public List<Expense> Expenses { get; set; }
+        public List<Transfer> Transfers { get; set; }
 
         public DateTime CreatedOn { get; set; }
         public string Notes { get; set; }
         public string Status { get; set; }
 
+        [Display(Name = "Sloth Transaction Id")]
+        public Guid? SlothTransactionId { get; set; }
```

Comment:
@jSylvestre i think it might be better to have this a string and not a guid.  It does happen that sloth transaction IDs are GUID but we are just treating it as if it was a string anyway and in the future the IDs could change if we use a different transaction system.  just seems like tying it to GUID is extra constraint for no benefit